Calendar/Get-CalendarDiagnosticObjectsSummary.ps1

```diff
@@ -188,6 +186,7 @@ function GetCalendarDiagnosticObjects {
     }
 
     Write-Host "Found $($CalLogs.count) Calendar Logs for [$Identity]"
+    $script:IsOrganizer = SetIsOrganizer($CalLogs)
```

Comment:
I am not seeing where this variable is being used `$script:IsOrganizer`

Reply:
Now on line 1610.

Reply:
This isn't being used still as you are resetting the value to `$IsOrganizer`. 

https://github.com/microsoft/CSS-Exchange/pull/1979/files#diff-c0a17053cc97c9900333ba5ee8350f54a57850310cf6cb5bf2ed9e64fec2de2aR1611

Reply:
Removed!